srcpkgs/libsigc++/template

```diff
@@ -1,16 +1,21 @@
 # Template file for 'libsigc++'
 pkgname=libsigc++
-version=2.10.3
+version=2.10.8
 revision=1
-build_style=gnu-configure
-configure_args="--enable-static --disable-documentation --disable-benchmark"
-hostmakedepends="m4"
+build_style=meson
+configure_args="-Dbuild-documentation=false -Dbenchmark=false -Dwarnings=max"
```

Comment:
There was no `configure` script in the 2.10.8 tarball, so I thought I might as well change the template to use Meson instead. (I could also switch it to Autotools, if you want).